nrt/data/SMIB/SMIB_BasicTestCases/SMIB_1_StepPm_IIDM/SMIB.jobs

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+    Copyright (c) 2022, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools
+    for power systems.
+-->
+<dyn:jobs xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:job name="SMIB_StepPm_IIDM">
+      <dyn:solver lib="dynawo_SolverIDA" parFile="solvers.par" parId="5"/>
+      <dyn:modeler compileDir="outputs/compilation">
+        <dyn:network iidmFile="SMIB.iidm" parFile="SMIB.par" parId="7"/>
```

Comment:
Can we remove the parameters here?

Reply:
The parFile and parId are required in the xsd. Maybe we can open a new issue to make them optional ?

Reply:
ok